python/pyarrow/tests/test_parquet.py

```diff
@@ -32,29 +32,21 @@
 
 import pandas.util.testing as tm
 
+pq = pytest.importorskip('pyarrow.parquet')
```

Comment:
Please revert this. See https://github.com/apache/arrow/commit/8f2b44b897b7083ee2a296c70397dc2d7d21d95e#diff-3a10a971f558573678baea521d62790a